### DISCUSSION TITLE
want's to learn python 🐍

### DISCUSSION THREAD

**Discussion by LoftyDroid** [community/community#37811] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi everyone,

I have been looking for resources for python programming/projects but, I'm not sure where to start.

I appreciate it If you can share some good repos.

**Comment by LUIS9799** [comment]
Hi!, You can learn basics at [W3schools](https://www.w3schools.com/) and then if you want to try out more and practice you can use [Replit](https://replit.com) that also provides very cool tutorials and let you practice everywere!!!, [Realpython](https://github.com/realpython) here you can find some materials and finally I recommend you read the python docs [PythonDocs](https://docs.python.org/3/)


Hope i helped

**Reply by LoftyDroid** [reply]
Thank you for sharing this.

**Comment by jfullstackdev** [comment, marked as answer]
I just search it for you on GitHub plus you have the perks of contributing and it will appear on your profile 

happy browsing

https://github.com/topics/school-project?l=python

https://github.com/topics/educational-project?l=python

https://github.com/topics/python-education

add note : don't you know that you can get a job using your GitHub profile ? I did it myself, so you want to build or contribute to other repos too plus there is the GitHub Sponsors, so others can sponsor you if they feel like it

**Reply by LoftyDroid** [reply]
Thank you for the answer.
